src/sdk/models/connections.ts

```diff
@@ -15,12 +15,18 @@ export interface Connection extends IEntityIdentifier<string> {
 	features?: ConnectionFeatures;
 	platform?: StoragePlatform;
 	account?: Account;
+	pool?: ConnectionPool;
 }
 
 export interface Account extends IEntityIdentifier<string> {
 	email?: string;
 }
 
+export interface ConnectionPool extends IEntityIdentifier<string> {
+	id?: string;
```

Comment:
This property shouldn't be required since it is inherited from `IEntityIdentifier<string>`.